cmd/tidb-lightning/main.go

```diff
@@ -38,16 +36,7 @@ func setGlobalVars() {
 func main() {
 	setGlobalVars()
```

Comment:
Should we remove this function? The `PreparedPlanCache` feature has some extra cost but seems we never rely on this feature.